routes/index.js

```diff
@@ -11,6 +11,7 @@ const apiKey = process.env.TOKBOX_API_KEY;
 const secret = process.env.TOKBOX_SECRET;
 
 const captionsUrl = 'https://api.opentok.com/v2/project';
+let captionsId; 
```

Comment:
It might make sense to add a comment clarifying that this approach only works with single room applications. We could also possibly set up a roomToSessionIdDictionary lookup table for captionsId instead. Let's see what @conshus thinks.

Reply:
This is being used for caching the call to `POST /captions/start`. However, having the `captionsId` defined here is going to cause issues since it is never cleared out. You should use a different method for caching this value.

Reply:
@manchuck, I changed the `/captions/stop` route, where after we successfully stopped the captions, I set the value of `captionsId` to an empty string. does that approach work for you?